frappe/utils/error.py

```diff
@@ -85,6 +86,7 @@ def log_error_snapshot(exception: Exception):
 	try:
 		log_error(title=str(exception), defer_insert=True)
 		logger.error("New Exception collected in error log")
+		add_data_to_monitor(exception=str(exception.__class__.__name__))
```

Comment:
Shouldn't need to cast to string right?

Reply:
yep, forgot to remove after using __name__